deploy/databases-development.json

```diff
@@ -16,7 +16,7 @@
                     "id": "[parameters('databaseName')]"
                 },
                 "options": {
-                    "x-ms-offer-throughput": "400"
```

Comment:
this is the wrong place to fix this.  This is a generated file. please fix it in [here](https://github.com/Azure/ARO-RP/blob/master/pkg/deploy/generator/resources.go#L1151)

Reply:
Please see the iteration, do I need undo the change in database-development.json?  And how to generate this file from /resources.go?

Reply:
running `make generate` will generate the json and bindata.go with new values. Our CI catches and blocks when some files haven't been regenerated:

https://msazure.visualstudio.com/AzureRedHatOpenShift/_build/results?buildId=29544242&view=logs&j=33dace58-8e3b-5448-d29b-15d14274b17c&t=5a927627-0f11-5c70-b682-973be05e7fa3&l=1380